core-bundle/contao/themes/flexible/styles/main.css

```
@@ -729,6 +725,59 @@ h2.sub_headline {
 	margin-left: 1%;
 }
 
+/* Jump targets */
+.jump-targets {
+	min-height: 30px;
+	padding-top: 1px;
+	background: var(--panel-bg);
+	border-bottom: 1px solid var(--content-border);
+	position: sticky;
+	top: 0;
+	z-index: 3;
+}
+
+.jump-targets .inner {
+	overflow-x: scroll;
+}
```

```suggestion
	scrollbar-width: none;
}

.jump-targets .inner::-webkit-scrollbar {
	display: none;
}
```

I think we should hide the horizontal scrollbar.